functions/src/logic/buildQueue/cleaner.ts

```diff
@@ -42,34 +42,30 @@ export class Cleaner {
       // @see https://docs.github.com/en/actions/learn-github-actions/usage-limits-billing-and-administration
       const ONE_HOUR = 1000 * 60 * 60;
       const sixHoursAgo = new Date().getTime() - 6 * ONE_HOUR;
-
-      const { seconds, nanoseconds } = lastBuildStart;
-      const buildStart = new Timestamp(seconds, nanoseconds).toDate().getTime();
+      const buildStart = new Date(lastBuildStart.seconds * 1000).getTime();
```

Comment:
Should we extract a constant for `lastBuildStart.seconds * 1000`?

Reply:
Personally I don't think creating a const called `buildStartInSeconds` will really add much to the readability of the code.

Maybe I misunderstand the reason you're suggesting this.